hyperspy/drawing/utils.py

```diff
@@ -18,6 +18,9 @@
 
 import copy
 import itertools
+import textwrap
+from traits import trait_base
+from mpl_toolkits.axes_grid1 import make_axes_locatable
```

Comment:
Moving the hyperspy imports to here caused errors when importing `hspy`, so I've moved what I could up here, and left the others in the function.